xml/OptionPanel.xml

```diff
@@ -1,108 +1,182 @@
-        <Text class="group-header">FAN-MADE ACCESSORIES</Text>
-      </Panel>
-      <!-- Option: show token arranger -->
-      <HorizontalLayout class="group-content">
-        <VerticalLayout class="text-column">
-          <Text class="option-header">Show "Token Arranger"</Text>
-          <Text class="description">See the contents of the chaos bag at a glance! This tool displays a sorted table of the tokens to allow easier guessing of your odds.</Text>
-        </VerticalLayout>
-        <Toggle id="showTokenArranger" onValueChanged="onClick_toggleOption(showTokenArranger)"/>
-      </HorizontalLayout>
+          <!-- Option: use clickable clue-counters -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Use clickable clue-counters</Text>
+                <Text class="description">Instead of automatically counting clues in the respective area on your playermat, this displays a clickable counter for clues.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="useClueClickers" onValueChanged="onClick_toggleOption(useClueClickers)"/>
+            </Cell>
+          </Row>
 
-      <!-- Option: show clean up helper -->
-      <HorizontalLayout class="group-content">
-        <VerticalLayout class="text-column">
-          <Text class="option-header">Show "Clean Up Helper"</Text>
-          <Text class="description">Useful for campaign-play: It resets play areas to allow continuous gameplay in the same savegame.</Text>
-        </VerticalLayout>
-        <Toggle id="showCleanUpHelper" onValueChanged="onClick_toggleOption(showCleanUpHelper)"/>
-      </HorizontalLayout>
+          <!-- Group: fan-made accessories -->
+          <Row class="group-header">
+            <Cell class="group-header">
+              <Panel class="group-header" image="option_header2">
+                <Text class="group-header">FAN-MADE ACCESSORIES</Text>
+              </Panel>
+            </Cell>
+          </Row>
 
-      <!-- Option: show hand helper -->
-      <HorizontalLayout class="group-content">
-        <VerticalLayout class="text-column">
-          <Text class="option-header">Show "Hand Helper"</Text>
-          <Text class="description">Never count your hand cards again! This tool does that for you and can even take "Dream-Enhancing Serum" into account. Also includes a button for randomly discard a card.</Text>
-        </VerticalLayout>
-        <Toggle id="showHandHelper" onValueChanged="onClick_toggleOption(showHandHelper)"/>
-      </HorizontalLayout>
+          <!-- Option: show token arranger -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Token Arranger"</Text>
```

Comment:
I think we can get rid of the repetitive "Show" throughout these, and the quotes around the item.  Just Token Arranger (et al) should do the job, the showing/enabling is implied with the checkbox.

Reply:
Yeah, we could do that. I was cautious of unclear wording to make sure that everyone understands that this spawns an additional object. It's not unthinkable to have a tool that does not have an interactable object.

Reply:
That's actually one of the reason I think we should get rid of "show".  As far as the options go you're just enabling or disabling something.  What form that takes shouldn't be relevant, though I can see both sides.  Going to resolve this and leave it up to you.